### PR TITLE
Feature: Tier Boosted Pet Display

### PR DESCRIPTION
**Pull request by Thunderblade73**
image

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/config/features/misc/pets/PetConfig.java [hunk 1]
```diff
@@ -28,4 +28,10 @@ public class PetConfig {
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean hideAutopet = false;
+
+    @Expose
+    @ConfigOption(name = "Show Tier Boost", desc = "Shows an §c● §7by a pet when an tier boost core is equipped")
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/misc/pets/PetConfig.java]: Maybe it's just me but "an" sounds funny I think both could just be "a". Also missing the fullstop at the end of the description

File: src/main/java/at/hannibal2/skyhanni/config/features/misc/pets/PetConfig.java [hunk 2]
```diff
@@ -28,4 +28,10 @@ public class PetConfig {
...
+    @ConfigEditorBoolean
+    @FeatureToggle
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/misc/pets/PetConfig.java]: I don't feel this feature qualifies for this annotation but up to you

**Review by CalMWolfs — CHANGES_REQUESTED** [review]
Pr looks fine apart from those two things. Which prob also applies to the other pr of yours that is very similar but I won't rereview that. Though I do have a suggestion, instead of having two features like this with similar logic what if you made a drop-down list for the user to select which pet items they want to display a symbol for. This would make adding new ones much easier and stop some code duplication but this is up to you

**Comment by Thunderblade73** [comment]
merged with #850

**Pull request closed without merging**